src/Select.js

```diff
@@ -562,6 +562,32 @@ export default class Select extends Component<Props, State> {
     });
   }
 
+  changeFocusedOption(opt: string){
```

Comment:
Why not just `focusOption` ?

Reply:
There is a method already with that name that changes the focused option based on direction (up, down, last, etc)